### DISCUSSION TITLE
Yank selection does not include last character

### DISCUSSION THREAD

**Discussion by valentingregoire** [JetBrains/ideavim#576] [Q&A] [answered]
Suppose the following piece of Python code:
```python
class SomeUnitTest(TestCase):
    pass
```
Whenever you would want to yank the `TestCase`, one would do `yi(`. However, in some cases, it is useful to have a selection before yanking like so: `vi(y`. Ultimately, the selection would be yanked. This results in `TestCas` being yanked (without the last `e`). In normal (neo)vim, that is not the case. Is this a bug or is this configurable? If configurable, how can I change this to match the default vim behaviour?

**Comment by AlexPl292** [comment]
Hi, this is not expected behaviour. Could you please share your `~/.ideavimrc` ?

**Reply by valentingregoire** [reply]
In the meantime, I have found the answer after looking over my `.ideavimrc` again. I must have looked over the `set selection=exclusive` line in my first iteration 🙈. Thanks!

**Comment by valentingregoire** [comment, marked as answer]
The option `selection=exclusive` was the root cause of the behavior. Changing it to (the default) `inclusive` solved this.
